app/views/tasklists/_tasks.html.haml

```diff
@@ -0,0 +1,22 @@
+.tab-content
+  -@tasks.each do |task|
+    .row
+      .card.col-sm-12
+        .row.card-body{id: "card#{task.id}"}
+          .col-sm-1.align-self-center
+            %input{type: 'checkbox', id: "checkbox#{task.id}", onclick: "updateTaskStatus(#{params[:id]},#{task.id})", "checked": task.status}
+          .col-sm-8.align-self-center
+            %div{id:"task#{task.id}",onClick:"renderUpdate(#{params[:id]},#{task.id})"}
+              =task.name
+          .col-sm-1.align-self-center
+            %span.fa.fa-edit{onclick: "renderUpdate(#{params[:id]},#{task.id})", id: 'taskUpdate'}
+          .col-sm-1.align-self-center
+            %span.btn.far.fa-trash-alt{onClick:"deleteTask(#{params[:id]},#{task.id})", id: 'taskDelete'}
```

Comment:
I am getting tired of rewriting this suggestion

Reply:
done